src/macros.rst

```diff
@@ -980,49 +980,25 @@ When transcribing a metavariable
 A metavariable indication in a macro repetition in transcription shall be
 transcribed to the matched tokens in order,
```

Comment:
I feel that the sentence is unfinished. Perhaps this will make it read better:

A metavariable indication in a macro repetition in transcription shall be transcribed to the matched tokens in order, **as follows**: